source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRShowCardTarget.mm

```diff
@@ -84,6 +84,8 @@ - (void)createShowCard:(NSMutableArray *)inputs superview:(UIView<ACRIContentHol
     CGFloat showCardPadding = [_config getHostConfig]->GetSpacing().paddingSpacing;
 
     _adcView.backgroundColor = UIColor.clearColor;
+    
+    _button.accessibilityValue = NSLocalizedString(@"card collapsed", nil);
```

Comment:
nit: we could create a string variable to share between here and line 131: NSString *hint = hidden ? @"card expanded" : @"card collapsed";

Reply:
Considering are different functions (initialization and toggle), and is a local variable just used once, could be better to be maintained as local, but I will check it.